### PR TITLE
source: add page

### PR DESCRIPTION
**Pull request by Aracki**
Add `source` page.

### CODE REVIEW

File: pages/common/source.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# source
+
+> Evaluate contents of file.
```
Comment (mfrw) [inline pages/common/source.md]: How about `A shell builtin command to executes the content of the file passed as argument`

File: pages/common/source.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+- Causes fish to re-read its initialization file:
```
Comment (mfrw) [inline pages/common/source.md]: This would only work if we are in `fish` shell. I think we should mention that.

File: pages/common/source.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# source
+
+> Evaluate contents of file.
+
+- Cause fish to re-read its initialization file:
```
Comment (agnivade) [inline pages/common/source.md]: We need an example which shows a basic usage. Something like source {{file.sh}} would do.

File: pages/common/source.md (hunk 1 above)
Comment (Aracki) [reply pages/common/source.md]: There is another explanation from bash shell: `Execute commands from a file in the current shell.` What do you think about that?

File: pages/common/source.md (hunk 3 above)
Comment (Aracki) [reply pages/common/source.md]: How you would change the example of mine: `source ~/.config/fish/config.fish`?

File: pages/common/source.md (hunk 1 above)
Comment (sbrl) [reply pages/common/source.md]: I like the latter there :-)

File: pages/common/source.md (hunk 3 above)
Comment (sbrl) [reply pages/common/source.md]: This isn't a `fish`-specific command - `bash` has it too. Perhaps we could adapt the description here a bit?

Comment (Aracki) [reply pages/common/source.md]: @sbrl I have added two more commands. And changed description too.

File: pages/common/source.md (hunk 2 above)
Comment (Aracki) [reply pages/common/source.md]: I've added bash examples alongside fish. Do you think it's enough?

File: pages/common/source.md (hunk 3 above)
Comment (sbrl) [reply pages/common/source.md]: I still see `Cause fish to re-read its initialization file` here. Was it a different description you changed?

Comment (sbrl) [reply pages/common/source.md]: Sounds good to me! A generic `{{path/to/file}}` example is all that's needed I think.

Comment (Aracki) [reply pages/common/source.md]: Maybe `Evaluates contents of file` or `Execute commands from a file` (same as main description)?

Comment (agnivade) [reply pages/common/source.md]: `Evalute contents of a given file` sounds good.

**Comment by sbrl** [comment]
Merging as @mfrw's comments have been resolved.

**Pull request merged**